### PR TITLE
Added missing check

### PR DESCRIPTION
**Pull request by NonSwag**
Fixed the option "spigot.yml > commands > log" so commands won't be logged into the console anymore if the option is disabled.
The respective code in ServerGamePacketListenerImpl was missing a simple check whether the option is disabled.

### CODE REVIEW

**Comment by Owen1212055** [comment]
It seems you have two patches in your PR, please remove your previous patch.

File: patches/server/0954-added-missing-check.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
++import org.spigotmc.SpigotConfig;
+ // CraftBukkit end
+ 
+ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, TickablePacketListener, ServerGamePacketListener {
+@@ -2256,7 +2257,8 @@ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, Tic
+     private void performChatCommand(ServerboundChatCommandPacket packet) {
+         // CraftBukkit start
+         String command = "/" + packet.command();
+-        ServerGamePacketListenerImpl.LOGGER.info(this.player.getScoreboardName() + " issued server command: " + command);
++        if (SpigotConfig.logCommands) // Paper - option was not respected
```
Comment (Owen1212055) [inline patches/server/0954-added-missing-check.patch:24]: Make sure you wrap your diff in paper comments.
Also use a fully qualified import.
Don't change the indentation of the logging message, this allows you to reduce the diff.
```java
if (org.spigotmc.SpigotConfig.logCommands) // Paper - option was not respected
ServerGamePacketListenerImpl.LOGGER.info(this.player.getScoreboardName() + " issued server command: " + command);
} // Paper
```

**Review by electronicboy — APPROVED** [review]

File: patches/server/0954-added-missing-check.patch [hunk 2]
```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Thu, 8 Dec 2022 20:25:05 +0100
+Subject: [PATCH] added missing check
```
Comment (Machine-Maker) [inline patches/server/0954-added-missing-check.patch:4]: The patch title needs to be more descriptive. This tells someone almost nothing.

File: patches/server/0953-fixed-npe.patch [hunk 3]
```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Tue, 6 Dec 2022 23:04:21 +0100
+Subject: [PATCH] fixed npe
```
Comment (Machine-Maker) [inline patches/server/0953-fixed-npe.patch:4]: This belongs in a separate PR. You seemed to have closed your other PRs for this?

**Review by Machine-Maker — CHANGES_REQUESTED** [review]
Also Owen's comment.

**Comment by Owen1212055** [comment]
Superseded as inactivity has prevented us from pushing these patches. Thank you for your contributions!

**Pull request closed without merging**